### PR TITLE
Nairi Brown HW1

### PR DESCRIPTION
**Pull request by nbrown20**
In this homework, I wrote an introduction. I used the different features of R such as headers, bullets, and inserting images. The TAs were super helpful for inserting images! I was having trouble because I uploaded the file to GitHub but I hadn't pulled the changes into my repo. Other than that, it was fairly straightforward.

### CODE REVIEW

**Review by gilheany — COMMENTED** [review]
I love your headers on both ends! For some reason, the picture did not show up for me. I also had problems with inserting the image, it might just be my computer. I like the bullets that you used!

File: README.md [hunk 1]
```diff
@@ -1 +1,24 @@
 # hw01
+
+# INTRODUCTION
+My name is Nairi Brown. I am from Massachusetts. I also went to UChicago for undergrad. 
+At home, I have a wonderful dog named **Louie** who is adorable. 
+
+*see picture below*
+
+![](lou.jpeg)
```
Comment (jnseddon) [inline README.md:9]: Your dog is very cute! Not to repeat our professor, but perhaps he could have been scaled down a little (not that I know how to do that...)

File: README.md [hunk 2]
```diff
@@ -1 +1,24 @@
...
+
+Here are some things that are cute about Louie:
+
+* everything
```
Comment (jnseddon) [inline README.md:13]: I like this list!

File: README.md [hunk 3]
```diff
@@ -1 +1,24 @@
...
+  * sorry if that was a let-down of a list
+
+*I've embedded a link below with more dog breeds. Yay! Dogs!*
+
+[Link to More Dogs](https://www.akc.org/dog-breeds/)
+
+
+I realize that this has been more of an intro to my dog, but I hope it was enjoyable.
```
Comment (jnseddon) [inline README.md:21]: Indeed, it was enjoyable!

File: README.md (hunk 1 above)
Comment (jnseddon) [inline README.md:9]: Your dog is very cute! Not to repeat the professor, but perhaps you could have shrunk the image down a little.

**Review by jnseddon — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   +        |              |            |
| **Coding strategy**         |   +        |              |            |
| **Presentation**            |           |     0         |            |
| **Achievement, creativity** |           |        0      |            |
| **Reproducibility**         |        +   |              |            |

Remarks:
I like your introduction, and viewed it without problems, unlike the other commenter, I think. As mentioned, your dog is cute! I also had a little trouble with images, but definitely feel like figuring it out was also a learning experience!

File: README.md (hunk 1 above)
Comment (djanvier) [inline README.md:9]: Adorable!

Comment (djanvier) [reply README.md:9]: Resizing images in Markdown/GitHub can be a bit tricky, so I would recommend resizing the original images if you decide to go down that route

File: README.md (hunk 3 above)
Comment (djanvier) [inline README.md:21]: It was nice to meet them!

**Review by djanvier — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |           |       x       |            |
| **Presentation**            |           |       x       |            |
| **Achievement, creativity** |      x     |             |            |
| **Reproducibility**         |    x       |              |            |

- Good job! 😄

**Pull request closed without merging**